cub/cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -160,6 +160,29 @@ constexpr offset_size classify_offset_size()
   return sizeof(OffsetT) == 4 ? offset_size::_4 : sizeof(OffsetT) == 8 ? offset_size::_8 : offset_size::unknown;
 }
 
+template <class ValueT,
+          class AccumT,
+          class OffsetT,
+          op_type OpTypeT,
+          primitive_accum PrimitiveAccumulator = is_primitive_accum<AccumT>(),
+          offset_size OffsetSize               = classify_offset_size<OffsetT>(),
+          value_size ValueSize                 = classify_value_size<ValueT>()>
+struct sm75_tuning;
+
+template <class ValueT, class AccumT, class OffsetT>
+struct sm75_tuning<ValueT, AccumT, OffsetT, op_type::plus, primitive_accum::yes, offset_size::_8, value_size::_4>
+{
+  // ipt_7.tpb_128.ns_628.dcid_1.l2w_520.trp_1.ld_0
```

Comment:
Remark: I am missing the scores here, like `1.080133 1.000000 1.075577 1.248963`.